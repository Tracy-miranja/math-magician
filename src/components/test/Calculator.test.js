import React from 'react';

import renderer from 'react-test-renderer';
import Display from '../Calculator';

test('test the home component', () => {
  const component = renderer.create(
    <Display />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
