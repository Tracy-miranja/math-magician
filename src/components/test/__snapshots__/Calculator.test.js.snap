// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the home component 1`] = `
<section
  className="container"
>
  <div
    className="finaal"
  >
    <p
      className="para"
    >
      let us do some math!!
    </p>
    <div
      className="all"
    >
      <div
        className="screen"
      >
        0
      </div>
      <div
        className="Keys"
      >
        <button
          onClick={[Function]}
          type="button"
        >
          AC
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          +/-
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          %
        </button>
        <button
          className="Orange"
          onClick={[Function]}
          type="button"
        >
          รท
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          7
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          8
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          9
        </button>
        <button
          className="Orange"
          onClick={[Function]}
          type="button"
        >
          ร
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          4
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          5
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          6
        </button>
        <button
          className="Orange"
          onClick={[Function]}
          type="button"
        >
          -
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          1
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          2
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          3
        </button>
        <button
          className="Orange"
          onClick={[Function]}
          type="button"
        >
          +
        </button>
        <button
          className="Zero"
          onClick={[Function]}
          type="button"
        >
          0
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          .
        </button>
        <button
          className="Orange"
          onClick={[Function]}
          type="button"
        >
          =
        </button>
      </div>
    </div>
  </div>
</section>
`;
